src/yio-interface/integrationinterface.h

```diff
@@ -23,14 +24,17 @@
 #pragma once
 
 #include <QVariant>
+#include "integrationconnectionstates.h"
 
 // FIXME provide complete API documentation with all QML requirements of the implementation!
 /**
  * @brief The IntegrationInterface must be implemented by the integration plugins.
  */
 class IntegrationInterface {
  public:
-    enum States { CONNECTED = 0, CONNECTING = 1, DISCONNECTED = 2 };
+    // enum States { CONNECTED = 0, CONNECTING = 1, DISCONNECTED = 2, STANDBY = 3 };
+    typedef IntegrationConnectionStates::States States;
+
```

Comment:
Remove commented out code.
With an enum class use the full name and not a typedef. See remote-software.